bears/stylus/StylintBear.py

```diff
@@ -0,0 +1,36 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='stylint',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+):?(?P<column>\d+)?\s+.*'
```

Comment:
Due to the difference in output messages :
1. For mixed spaces and tabs, I get
```
3:0 mixed warning mixed spaces and tabs
``` 
2. For trailiing whitespaces, I get
```
2:13  warning  trailing whitespace  trailingWhitespace
```